script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -269,63 +323,80 @@ The smile hasn't faded from her face.
 [sha:cbda7178dd5580d00898dc1f2590615eb62d0d9a]{
 -- Page 444, Offset 19372.
 -- ねっとりと赤い糸を垂らす槍斧を持ち上げ、次の獲物の頭蓋骨に振り下ろす。
-She lifts her halberd, from which a sticky red thread hangs, and drops it down on the skull of her next prey.
+Sticky red threads cling to her halberd as she hoists it aloft again,
+ then brings it crashing down on the skull of her next target.
 }
 [sha:028e8b3efa481f0a94986b78ced5c72d8bb4259b]{
 -- Page 444, Offset 19373.
 -- 呼吸を乱しながら。
-Breathing heavily―――
+Her breathing is frenzied.
 }
 [sha:f4ff2fc67aff990d758e11f916786b821b095b85]{
 -- Page 444, Offset 19374.
 -- その一撃によって絞り出される悲鳴に興奮するように。
-As though excited by the shriek wrought out by that one blow.
+It's as though she's been driven wild with excitement by the scream that blow wrung from her victim.
```

Comment:
I agree that a straightforward tl of ように can feel boring, but something about that exact suggestion doesn't feel right. It feels like the sentence is missing a second half or something, but it's hard to put into words my exact issue with it. Rearranging it to 
> The scream wrung from the victim of her blow seems to have driven her wild with excitement.

makes it parse better to me, but I feel like it subtly changes the meaning - her breathing heavily _as though_ she's excited lends plausible deniability (she's breathing heavily because she's working hard, not because she's getting off on it), whereas "seems to have driven" reads as much more matter-of-fact to me.

Reply:
I think the way this sentence flows with the previous, it makes sense to leave it as I wrote it.
Had it been in isolation, then yeah I could see the argument for how you wrote it.

Reply:
Reading it again today it still feels off. If it were the same sentence as the one before, separated with a comma, I'd like it better, but as-is it feels weirdly disjoint. @Avuxo, third opinion?

Reply:
Then just add the ――― stuff

Reply:
I mean that just seems even clunkier to me, I hate splitting sentences across linebreaks like that. If you're desperate for `seems`, I'd be happy with 
> She seems like she's been driven wild with excitement by the scream that blow wrung from her victim.

if you just hate `as though`, then I'd also be fine with
> It's like she's been driven wild with excitement by the scream that blow wrung from her victim.

Reply:
I don't like either of the comparative ones, that's why I wanted to avoid it!
ように is a much more nuanced expression than that.